### ISSUE TITLE
Webhook handler not updating cancellation details on Firebase.

### ISSUE THREAD

**Issue by eddieespinal** [invertase/stripe-firebase-extensions#602]
If the user cancels a subscription on Stripe, the webhook doesn't update Firebase with that information. The fields cancel_at_period_end and cancel_at are not getting updated correctly.

**Comment by jauntybrain** [comment]
Hi @eddieespinal!

As per [Stripe documentation](https://stripe.com/docs/api/subscriptions/object#subscription_object-canceled_at), if the subscription is manually canceled by the user, it will be reflected in the `canceled_at` value and won't impact the `cancel_at_period_end` and `cancel_at` values. 

In the example below, I canceled a subscription that was scheduled to be automatically canceled on Feb. 4th. As you can see, that change is reflected in the collection.

Please let me know if this resolves your issue.

<img width="428" alt="Screenshot 2024-01-28 at 1 33 57 PM" src="https://github.com/invertase/stripe-firebase-extensions/assets/105740958/f3c397e0-de10-4efe-8c63-4e3fe62957fd">

**Comment by eddieespinal** [comment]
Hi @jauntybrain 

Thank you for getting back to me. I tried that but I still don't see the values updating. See my screenshots below:

I cancelled the plan which shows Feb 2nd but if you see my Firestore DB, its not reflecting it.

<img width="967" alt="Screenshot 2024-01-29 at 9 23 48 PM" src="https://github.com/invertase/stripe-firebase-extensions/assets/201202/2b55efc0-65ba-4f53-b49b-8ce53304f124">

Firebase doesn't seems to be updating correctly.

<img width="488" alt="Screenshot 2024-01-29 at 9 28 44 PM" src="https://github.com/invertase/stripe-firebase-extensions/assets/201202/e0942dd8-a4c2-4680-9eec-1c3fe9abf2a6">

What did I missed?  Thanks again.

**Comment by eddieespinal** [comment]
OMG it was my fault. I went back to double check that I had correctly added each event and I noticed that I was missing the most important one, the **customer.subscription.updated**. That was the issue. Now is updating correctly. Thank you for helping me out.


<img width="489" alt="Screenshot 2024-01-29 at 9 49 02 PM" src="https://github.com/invertase/stripe-firebase-extensions/assets/201202/c02c74ab-09a3-4f47-adca-f628ba211818">
